tools/clang/test/CodeGenSPIRV/spv.inline.type.hlsl

```diff
@@ -0,0 +1,32 @@
+// RUN: %dxc -T ps_6_0 -E main -fcgl  %s -spirv | FileCheck %s
+
+// TODO(5751): enable Array test when `using` has been implemented
```

Comment:
#5751 is closed now, can these be uncommented?

Reply:
Hm, this still doesn't work completely. Something to do with how Texture2D is something of a special case, I think.

Reply:
I'm *really* not sure what's causing this. It seems to be an error from `SemaHLSL.cpp`, but it doesn't happen for non-SpirvType code (https://godbolt.org/z/KeoGMaEqs).

I've filed a bug at #6498. I'll reference the bug in this test.